ocean_lib/example_config.py

```diff
@@ -83,9 +83,9 @@
 
 
 @enforce_types
-def get_config_dict(chain_id: int, network_url: str) -> dict:
-    if chain_id not in CONFIG_NETWORK_HELPER:
-        raise ValueError("The chain id for the specific RPC could not be fetched!")
+def get_config_dict(network_url: str) -> dict:
```

Comment:
aren't there any usages of this function, or usages of the "CHAIN_ID" key anywhere? I remember at least in example config I left an ocurrence. Those should be removed as well.

Reply:
Yes, I saw the CHAIN_ID key in the config_helper dict. I will delete from there as well.